alacritty_terminal/src/term/mod.rs

```diff
@@ -40,8 +40,12 @@ const MAX_SEARCH_LINES: usize = 100;
 /// Default tab interval, corresponding to terminfo `it` value.
 const INITIAL_TABSTOPS: usize = 8;
 
-/// Minimum number of columns and lines.
-const MIN_SIZE: usize = 2;
+/// Minimum number of columns.
```

Comment:
Maybe we should add a new line between those 2 sentences?

Reply:
I like it.